docker-compose.yml

```diff
@@ -10,7 +10,7 @@
 version: "2"
 services:
     db:
-        image: mongo:3.2.6
+        image: freecodecamp/mongo:production
 
     server:
         image: node:8.9.0
```

Comment:
Might we want to do the same for node?

Reply:
Will it not be better to use named tags for LTS versions of both mongo and node?

Reply:
I would prefer named LTS images. One less thing to maintain. If you have a compelling reason to maintain our own images, I am always open for discussion.

Reply:
LTS would work for me!

I suggested using our own image as a ghetto key value store to maintain a blessed version. It would be as a one line Docker file, but would also allow for customisations if we want to.

However, I'm not married to that idea, and hopefully we'll only share this db with multiple projects for a short while.

Happy to use LTS / Stable instead!

For Mongo, 3.6 seems to be the current stable release (https://hub.docker.com/_/mongo/), for Node, 8.10.